hi-ml-cpath/src/health_cpath/scripts/generate_crossval_report.py

```diff
@@ -62,7 +69,8 @@ def generate_html_report(parent_run_id: str, output_dir: Path,
     if num_classes > 1:
         base_metrics_list = [MetricsKey.ACC, MetricsKey.AUROC]
```

Comment:
the lists for metrics need to be updated for both binary and multi-class cases

Reply:
Good point, done!